src/sankey.js

```diff
@@ -191,6 +197,39 @@ export default function Sankey(config) {
     }
   }
 
+  /**
+   * Compute nodes with fixed position
+   * @param nodes
+   * @returns {any[]}
+   */
+  function computeFixedNodeLayer({nodes}) {
+    const d3group = group(nodes, d => d.dimIndex);
```

Comment:
rename please to something like `groupedNodes`

Reply:
changed